apps/04TowersOfHanoi.js

```diff
@@ -10,36 +10,71 @@ var stacks = {
     c: []
 };
 
+var moveCount = 0;
+
 function printStacks() {
     console.log("a: " + stacks.a);
     console.log("b: " + stacks.b);
     console.log("c: " + stacks.c);
 }
 
-function movePiece() {
-    // Your code here
+function restartGame() {
+    moveCount = 0;
+    var stacks = {
+    a: [4, 3, 2, 1],
```

Comment:
Remember to keep an eye on formatting, these lines setting up the stack should be indented.